### PR TITLE
Updated to support 2020.1 version of Resharper and Rider

### PR DESCRIPTION
**Pull request by mjb501**
Updated to support the 2020.1 version of Resharper and Rider. I need to add a couple of extra nuget references to JetBrains.Annotations, JetBrains.Lifetimes & Nuint as well as updating the Resharper & Rider references.

### CODE REVIEW

File: AsyncConverter.Tests/AsyncConverter.Rider.Tests.csproj [hunk 1]
```diff
@@ -30,9 +30,15 @@
     <ProjectReference Include="..\AsyncConverter\AsyncConverter.Rider.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="JetBrains.Rider.SDK.Tests" Version="2019.3.0" />
+    <PackageReference Include="JetBrains.Annotations" Version="2020.1.0" />
+    <PackageReference Include="JetBrains.Lifetimes" Version="2020.1.3" />
```
Comment (orenleto) [inline AsyncConverter.Tests/AsyncConverter.Rider.Tests.csproj:34]: What purpose of this package reference and one up?

File: AsyncConverter.Tests/AsyncConverter.Rider.Tests.csproj [hunk 2]
```diff
@@ -30,9 +30,15 @@
...
+    <PackageReference Include="JetBrains.Rider.SDK.Tests" Version="2020.1.0">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
     <PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="16.3.0" />
     <PackageReference Include="Microsoft.TestPlatform.TranslationLayer" Version="16.3.0" />
+    <PackageReference Include="NUnit" Version="3.10.1" />
```
Comment (orenleto) [inline AsyncConverter.Tests/AsyncConverter.Rider.Tests.csproj:41]: The project doesn't use NUnit.

**Review by orenleto — COMMENTED** [review]
I think that you need fix file [plugin.xml](https://github.com/BigBabay/AsyncConverter/blob/master/Rider/AsyncConverter.Rider/META-INF/plugin.xml):
* update idea-version to 201
* update minor app version

**Comment by BigBabay** [comment]
Need update nuget package version in AsyncConverter.csproj and  AsyncConverter.Rider.csproj

**Comment by BigBabay** [comment]
already fixed

**Pull request closed without merging**